velox/exec/benchmarks/ExchangeBenchmark.cpp

```diff
@@ -457,6 +457,15 @@ int main(int argc, char** argv) {
             << "deep10k: " << deep10kCounters.toString() << std::endl
             << "deep50: " << deep50Counters.toString() << std::endl
             << "struct1k: " << struct1kCounters.toString() << std::endl;
-  return 0;
+
+  // Deallocate ExchangeBenchmark and vectors before MemoryManager destruction
```

Comment:
Can we put a scope or add a function say runBenchmarks() so that we don't have to reset each individual local testing object? Thanks!